### PR TITLE
Added new Marketplace contribution page/section

### PR DESCRIPTION
**Pull request by cwarnermm**
Publishing general information for contributors about the types of contributions accepted into the web marketplace and what criteria must be met to be considered.

### CODE REVIEW

**Review by wiersgallak — APPROVED** [review]

File: site/content/integrate/marketplace-submissions/_index.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+title: "Contribute to the Marketplace"
+heading: "Contribute to the Marketplace"
+description: "If you’ve built an integration, playbook, or tool which supports Mattermost, you may want to consider contributing it to the community via the Mattermost Web Marketplace."
+weight: 100
+aliases: /extend/plugins/community-plugin-marketplace/
+---
+
+The [Mattermost web Marketplace](https://mattermost.com/marketplace/) supports discovery, installation, and updates of extensions of the Mattermost platform. If you have built a plugin, app, playbook, integration, or tool that helps other users get more from Mattermost, the web Marketplace is a great way to get feedback on your contribution and help make it more popular. Once your submission is accepted to the Web Marketplace, Mattermost will also send you swag!
```
Comment (azigler) [inline site/content/integrate/marketplace-submissions/_index.md:9]: Is the capitalization "Web Marketplace" or "web Marketplace" ?

File: site/content/integrate/marketplace-submissions/_index.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+We will also reach out to you to learn more about the integration, your experience developing with Mattermost, and to coordinate postings for social media. 
+
+## How to submit your contribution for consideration for the in-product marketplace
+
+There are additional functional, technical, and security requirements for apps and plugins to be considered for the in-product marketplace. You can check off these items when you submit your contributions by opening an [issue in our in-product marketplace repository](https://github.com/mattermost/mattermost-marketplace/issues/new?assignees=hanzei&labels=Plugin%2FNew&template=add_plugin.md&title=Add+%24REPOSITORY_NAME+to+Marketplace). 
+
+### Technical requirements
+
+1. The plugin or app works for 60,000 concurrent connections and within a High Availability deployment. **Note:** There are currently no publicly-available tools available to verify these properties. As such, they are checked during code review by a developer.
```
Comment (azigler) [inline site/content/integrate/marketplace-submissions/_index.md:39]: Is there any guidance on how a "High Availability deployment" is defined?

File: site/content/integrate/marketplace-submissions/_index.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+weight: 100
+aliases: /extend/plugins/community-plugin-marketplace/
+---
+
+The [Mattermost web Marketplace](https://mattermost.com/marketplace/) supports discovery, installation, and updates of extensions of the Mattermost platform. If you have built a plugin, app, playbook, integration, or tool that helps other users get more from Mattermost, the web Marketplace is a great way to get feedback on your contribution and help make it more popular. Once your submission is accepted to the Web Marketplace, Mattermost will also send you swag!
+
+All contributions are eligible to be added to our web marketplace at mattermost.com/marketplace after a review from our team. Apps and plugins that have been thoroughly reviewed by Mattermost will be eligible for our in-product marketplace as well with some additional requirements. Our in-product Marketplace enables customers to install and manage their apps and plugins from directly within the product. 
+
+## Type of integrations to submit to the web marketplace
```
Comment (azigler) [inline site/content/integrate/marketplace-submissions/_index.md:13]: "Types of integrations"

File: site/content/integrate/marketplace-submissions/_index.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+- Plugins
+- Webhook configuration
+- Playbook templates
+- Utility tools, such as importers, command line interfaces, and scripts
+
+## How to submit your contribution for consideration for the web marketplace
+
+Please fill out the information in [this form](https://forms.gle/PE8kmfSuneP9GWnq8) to be reviewed by a member of the Mattermost team. 
+
+Every contribution will go through the following checklist before being added to the web marketplace at mattermost.com/marketplace:
```
Comment (azigler) [inline site/content/integrate/marketplace-submissions/_index.md:25]: Couldn't "mattermost.com/marketplace" just be a hyperlink on "web marketplace" (or however we're going to capitalize it)?

File: site/content/integrate/marketplace-submissions/_index.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+1. Security reviews do not reveal any exploitable vulnerabilities.
+2. An email address or a username on the [Community Server](https://community.mattermost.com) is provided to be used to report vulnerabilities in the future.
+
+### Functional requirements
+
+1. The app or plugin must set a `min_server_version` in the manifest.
+2. The app or plugin must work on all Mattermost versions greater than or equal to the `min_server_version`.
+
+Please note that Mattermost reserves the right to reject any plugin submission from the Marketplace.
+
```
Comment (azigler) [inline site/content/integrate/marketplace-submissions/_index.md:53]: This is missing [Product and Documentation requirements](https://github.com/mattermost/mattermost-marketplace/blob/master/.github/ISSUE_TEMPLATE/add_plugin.md). To reduce duplicity, I think these sections above could be removed and we could link to the issue template *.md instead.

**Review by azigler — APPROVED** [review]
Suggesting a few alterations to reduce duplicity, but otherwise I'm just wondering about how "web marketplace" should be capitalized? We should standardize it. There's nothing that's blocking this from going live on my end, though. Thanks @cwarnermm!

**Comment by cwarnermm** [comment]
@neflyte - Are you open to taking over this developer documentation PR I started a few months ago? This PR contains outstanding reviewer feedback that hasn't yet been incorporated, and it likely needs a full review before it can be published. Given how long ago this PR was created, it's likely also quite behind in terms of the master branch.

This PR intends to introduce a new section that describes the Mattermost Marketplace, the types of contributions accepted into the web marketplace, and what criteria must be met to be considered.

The Integrations Product Manager, Sandy, is an excellent person to chat with further about this new page.

File: site/content/integrate/marketplace-submissions/_index.md (hunk 2 above)
Comment (neflyte) [reply site/content/integrate/marketplace-submissions/_index.md:39]: I've removed this section per your suggestion below. :slightly_smiling_face:

**Comment by neflyte** [comment]
This PR has been replaced with #1154

**Comment by cwarnermm** [comment]
Closing this PR in favour of https://github.com/mattermost/mattermost-developer-documentation/pull/1154

**Pull request closed without merging**